.swiftlint.yml

```diff
@@ -0,0 +1,51 @@
+disabled_rules: # rule identifiers to exclude from running
+  - colon
```

Comment:
These are a lot of disables rules, would be nice if you enable them and fix or disabling locally depending on each case.